packages/flutter/lib/src/painting/_network_image_web.dart

```diff
@@ -11,6 +11,8 @@ import 'image_provider.dart' as image_provider;
 import 'image_stream.dart';
 
 /// The dart:html implemenation of [image_provider.NetworkImage].
+///
+/// NetworkImage on the web does not support decoding to a specified size.
```

Comment:
the docs here won't be read by dartdoc, as i understand it.

Reply:
Is there something that needs to be done to get this to work with dartdocs or is this just expected/know/accepted?

Reply:
just put the docs on the "public" NetworkImage